main.tf

```diff
@@ -1,19 +1,27 @@
 locals {
+<<<<<<< HEAD
   enabled                    = module.this.enabled ? 1 : 0
   parent_zone_record_enabled = var.parent_zone_record_enabled && module.this.enabled ? 1 : 0
   zone_name                  = local.parent_zone_record_enabled == 1 ? var.zone_name : replace(var.zone_name, ".$${parent_zone_name}", "")
+=======
+  enabled                    = module.this.enabled
+  parent_zone_record_enabled = local.enabled && var.parent_zone_record_enabled
+  public_zone                = try(length(var.private_hosted_zone_vpc_attachments), 0) == 0
+>>>>>>> kevcube/support_private_hosted_zone
```

Comment:
Merge issue

Reply:
My bad, happened when I was cleaning up the initial merge pull. Resolved.